src/traces/waterfall/hover.js

```diff
@@ -34,16 +38,16 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode) {
     if(!di.isSum) {
         // format delta numbers:
         if(size > 0) {
-            point.extraText = size + ' ' + DIRSYMBOL.increasing;
+            point.extraText = formatNumber(size) + ' ' + DIRSYMBOL.increasing;
```

Comment:
Use `Axes.hoverLabelText`, it's made specifically for this purpose:
https://github.com/plotly/plotly.js/blob/c6573487f33ffe02f38f443690e5fdb824fdb161/src/traces/box/hover.js#L170

Reply:
Thanks for the review @alexcjohnson.
Revised in https://github.com/plotly/plotly.js/pull/3711/commits/07b6c9c296b2037ebda7d12619e31354fc263296.